pulp_ansible/app/serializers.py

```diff
@@ -17,8 +17,13 @@ class AnsibleRoleSerializer(ContentSerializer):
         lookup_url_kwarg='role_pk',
     )
 
+    version_count = serializers.IntegerField(
+        source='versions.count',
+        read_only=True
+    )
```

Comment:
This change wasn't necessary. It just made some of the debugging easier and I thought it was useful so I left it.